progress/render.go

```diff
@@ -195,13 +195,15 @@ func (p *Progress) moveCursorToTheTop(out *strings.Builder) {
 }
 
 func (p *Progress) renderTracker(out *strings.Builder, t *Tracker, hint renderHint) {
-	message := t.message()
-	if strings.Contains(message, "\t") {
-		t.Message = strings.Replace(message, "\t", "    ", -1)
```

Comment:
FWIW: If this PR isn't accepted, this should be switched to UpdateMessage regardless

Reply:
I missed this instance in my previous change. 😞 

I think the better way to do this would be to get the message value one single time, and then use it for a single render event.